apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -102,7 +119,26 @@ async function deliverMessage (messageId) {
         return MESSAGE_BLACKLISTED_STATUS
     }
 
-    const { strategy, transports, isVoIP } = getMessageOptions(message.type)
+    const { strategy, transports, isVoIP, throttlePeriodForUser = null } = getMessageOptions(message.type)
+
+    if (throttlePeriodForUser) {
+        const throttlingCacheKey = getThrottlingCacheKey(message)
+        const lastCommonNewsItemSentDate = await throttlingCacheClient.get(throttlingCacheKey)
```

Comment:
I think `lastCommonNewsItemSentDate` should be renamed :)

Reply:
Renamed